cli/stream_command.go

```diff
@@ -2366,6 +2474,10 @@ func (c *streamCmd) validateCfg(cfg *api.StreamConfig) (bool, []byte, []string,
 }
 
 func (c *streamCmd) addAction(pc *fisk.ParseContext) (err error) {
+	if (c.subjectTransformSource != "" && c.subjectTransformDest == "") || (c.subjectTransformSource == "" && c.subjectTransformDest != "") {
```

Comment:
similar comment here as with the one that was in `editAction`, its action at a distance and would be clearer if added to `prepareConfig` I think